auction-server/src/auction/service/verification.rs

```diff
@@ -498,6 +499,64 @@ impl Service<Svm> {
         }
     }
 
+    fn validate_swap_transaction_instructions(
+        &self,
+        tx: &VersionedTransaction,
+    ) -> Result<(), RestError> {
+        tx.message
+            .instructions()
+            .iter()
+            .enumerate()
+            .try_for_each(|(index, ix)| {
+                self.validate_swap_transaction_instruction(tx.message.static_account_keys(), ix)
+                    .map_err(|e| {
+                        RestError::BadParameters(format!(
+                            "Invalid instruction at index {}: {:?}",
+                            index, e
+                        ))
+                    })
+            })?;
+
+        Ok(())
+    }
+
+    fn validate_swap_transaction_instruction(
+        &self,
+        accounts: &[Pubkey],
```

Comment:
is this compatible with account lookup table?

Reply:
don't need lookup table right now because all the accounts we care about are programs/signers

Reply:
got it